src/components/Main/index.js

```diff
@@ -1,29 +1,20 @@
 import React from 'react';
+import Score from '../Score';
 
 const Main = () => {
   return (
     <main className="Main">
-      <div className="temp-score">
-        <h3>Team 1 Score</h3>
-        <p>3</p>
-        <p>Lynne</p>
-        <p>Justin</p>
-      </div>
+      <Score team={1} score={5} players={["Lynne", "Justin"]}/>
```

Comment:
I like the dummy data that you are using in here currently. We'll have to convene with the backend to get a better Idea of what data we're going to need to pass where?